recipes/cram/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "cram" %}
+{% set version = "0.7" %}
+{% set bundle = "tar.gz" %}
+{% set hash_type = "md5" %}
+{% set hash = "2ea37ada5190526b9bcaac5e4099221c" %}
+{% set build = 0 %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.{{ bundle }}
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ bundle }}
+  {{ hash_type }}: {{ hash }}
+
+build:
+  number: {{ build }}
+  script: python setup.py install
+
+requirements:
+  build:
+    - python
+
+  run:
+    - python
+
+test:
+  imports:
+    - cram
+
+  commands:
+    - cram -h
+
+about:
+  home: https://bitheap.org/cram/
+  license: GPL-2.0
+  summary: 'A simple testing framework for command line applications'
+  license_family: GPL2
```

Comment:
Is the license not in the source tarball? If not, please raise an issue upstream.
See http://conda-forge.github.io/docs/meta.html#packaging-the-license-manually